src/definitions.ts

```diff
@@ -114,6 +121,11 @@ export interface CloudSettings {
   push?: PushOptions;
 
   /**
+   * Settings for native auth.
+   */
+  nativeAuth?: NativeAuthOptions;
```

Comment:
These should really be under `settings.auth`

For the web client ID thing, it can be `settings.auth.google.webClientId`